mule-migration-tool-e2e-tests/src/test/resources/e2e/gaps/withsampledatafiles/output/src/main/mule/withsampledatafiles.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+
+    <flow name="withsampledatafilesFlow">
+        <set-payload value="#[payload]" mimeType="application/xml" />
+
+        <ee:transform doc:name="Transform Message">

```

Comment:
Not sure if this is a valid test, by the directory name this should be reported as unsupported, but the error here is different. Maybe is a case we need to handle

Reply:
It's an unsupported attribute `metadata:id`, the error says more or less that:
`Custom types defined in Studio 6 are not migrated to Studio 7`

Reply:
Ok, I was just confused by the purpose of the test. I think I understand now